cypress/e2e/tests/pages/global-settings/banners.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { BannersPagePo } from '@/cypress/e2e/po/pages/global-settings/banners.po';
+import { SettingsPagePo } from '@/cypress/e2e/po/pages/global-settings/settings.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import BurgerMenuPo from '@/cypress/e2e/po/side-bars/burger-side-menu.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import { LoginPagePo } from '@/cypress/e2e/po/pages/login-page.po';
+
+const bannersPage = new BannersPagePo();
+const burgerMenu = new BurgerMenuPo();
+const loginPage = new LoginPagePo();
+
+const settings = {
+  bannerLabel:   'Rancher e2e',
+  textAlignment: {
+    original: 'Center',
+    new:      'Right'
+  },
+  fontSize: {
+    original: '14px',
+    new:      '20px'
+  },
+  textDecoration:  'Underline',
+  bannerTextColor: {
+    original: '#141419',
+    new:      '#f80dd8',
+    newRGB:   'rgb(248, 13, 216)',
+  },
+  bannerBackgroundColor: {
+    original: '#EEEFF4',
+    new:      '#ddd603',
+    newRGB:   'rgb(221, 214, 3)'
+  }
+
+};
+
+describe('Banners', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('can navigate to Banners Page', { tags: ['@globalSettings', '@adminUser', '@standardUser'] }, () => {
+    HomePagePo.goTo();
+
+    const productMenu = new ProductNavPo();
+
+    BurgerMenuPo.toggle();
+
+    burgerMenu.categories().contains(` Configuration `).should('exist');
+    const globalSettingsNavItem = burgerMenu.links().contains(`Global Settings`);
+
+    globalSettingsNavItem.should('exist');
+    globalSettingsNavItem.click();
+    const settingsPage = new SettingsPagePo('local');
+
+    settingsPage.waitForPageWithClusterId();
+
+    const bannersNavItem = productMenu.visibleNavTypes().contains('Banners');
+
+    bannersNavItem.should('exist');
+    bannersNavItem.click();
+
+    bannersPage.waitForPageWithClusterId();
+  });
+
+  it('can show and hide Header Banner', { tags: ['@globalSettings', '@adminUser'] }, () => {
+    bannersPage.goTo();
```

Comment:
do these need to be goTo?

Reply:
Nice catch, I missed this file. fixed in https://github.com/rancher/dashboard/pull/9458/commits/7112368f8cf9ac062e0b74643ed3fefd04b3b551